src/igvfd/schemas/changelogs/differentiated_tissue.md

```diff
@@ -0,0 +1,2 @@
+## Changelog for differentiated_tissue.json
+* renamed organoid.json to differentiated_tissue.json
```

Comment:
Same as above. See https://github.com/IGVF-DACC/igvfd/blob/IGVF-128-rm-parenthesis-temp-schema/src/igvfd/schemas/changelogs/treatment.md for example.